src/issuance.rs

```diff
@@ -653,6 +671,54 @@ mod tests {
         )
     }
 
+    fn generate_identity_point_asset_base() -> AssetBase {
+        let identity_point =
+            (pallas::Point::generator() * -pallas::Scalar::one()) + pallas::Point::generator();
+        AssetBase::from_bytes(&identity_point.to_bytes()).unwrap()
+    }
+
+    fn identity_point_asset_base_test_params(
+        note1_value: u64,
+        note2_value: u64,
+        finalize: bool,
```

Comment:
`finalize` is always false so lets remove it.

Reply:
Done.